### DISCUSSION TITLE
While using editOnClick feature of inline editing, The column editor is showing 'adasda' as placeholder.

### DISCUSSION THREAD

**Discussion by vickybml** [Power-Components/livewire-powergrid#1234] [Q&A] [answered]
How to change/remove default cell placeholder of powergrid while using editOnClick feature.
![powergrid](https://github.com/Power-Components/livewire-powergrid/assets/43170128/d71881d9-8c07-493f-b873-99e5ed096683)

**Comment by xX-MISSIT-Xx** [comment]
Hi, i have the same problem ...

**Comment by xX-MISSIT-Xx** [comment]
I have found the error. It is in the blades but in the pg-editable.js "vendor/power-components/livewire-powergrid/resources/js/components/pg-editable.js"

"adasda" is permanently entered there.

**Reply by luanfreitasdev** [reply, marked as answer]
Fixed: https://github.com/Power-Components/livewire-powergrid/releases/tag/v5.1.14
